db/chores/update_auctions_billable_data.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateAuctionsBillableData
+  def perform
+    auction_tock_mappings.each do |mapping|
+      auctions = Auction.where(billable_to: mapping[:auction_billable_to])
+      account = ClientAccount.find_by(tock_id: mapping[:tock_id])
+      auctions.each { |auction| auction.update(billable_to: account.to_s) }
+    end
+  end
+
+  private
+
+  def auction_tock_mappings
```

Comment:
we shouldn't have to in the future because people will create auctions with the correct billable data. This is just for our existing auctions